tests/infra_test.go

```diff
@@ -141,7 +141,7 @@ var _ = Describe("Infrastructure", func() {
 				&expect.BExp{R: "localhost:~#"},
 				&expect.BSnd{S: "sync\n"},
 				&expect.BExp{R: "localhost:~#"},
-			}, 10*time.Second)
```

Comment:
In general, OK. I used this value to be overcautious against false negatives on slow CI. I don't have any hard data to back any choice here, so your guess (if not please document why is not a guess) is good as mine.

Reply:
Oh that's something I didn't fully tested yet, wanted to check what this duration actually means - if this time includes the time it takes to interact with the pod then yeah, it might be risky to lower it that much

Reply:
Changed it back. Apparently it's not an execution time but rather a timeout - so considering the `dd` operation is fairly quick, we (probably) never get to this timeout anyway